File: examples/user/read/node/readAnAccountById.js
const { UserService } = require("micro-js-client/user");

// Read an account by id, username or email. Only one need to be specified.
async function readAnAccountById() {
  let userService = new UserService(process.env.MICRO_API_TOKEN);
  let rsp = await userService.read({
    id: "user-1",
  });
  console.log(rsp);
}

readAnAccountById();

File: examples/user/read/go/readAnAccountById.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/user"
)

// Read an account by id, username or email. Only one need to be specified.
func ReadAnAccountById() {
	userService := user.NewUserService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := userService.Read(&user.ReadRequest{
		Id: "usrid-1",
	})
	fmt.Println(rsp, err)
}
